test/net/sourceforge/kolmafia/request/FightRequestTest.java

```diff
@@ -2377,4 +2377,17 @@ public void doesNotLogCircadianFailures() {
       }
     }
   }
+
+  @Nested
+  class Yachtzee {
+    @Test
+    void canTrackPartyYachtCombats() {
+      var cleanups = new Cleanups(withLastLocation("Sunken Party Yacht"));
```

Comment:
you should add `withProperty("encountersUntilYachtzeeChoice ", 20)` to the cleanups so that the property gets reset after the test.

I don't think it actually matters here but it's a good habit to get into :) plus it makes the 20 -> 19 thing explicit